exporter/kafkaexporter/compression.go

```diff
@@ -0,0 +1,41 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kafkaexporter
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Shopify/sarama"
+)
+
+func ConfigureCompression(compression string, saramaConfig *sarama.Config) error {
```

Comment:
Does this need to be exported?  If so, it needs godoc describing its use.  It probably doesn't, though, and thus should be unexported.

Reply:
I also don't see why this would be exported, but I saw that `ConfigureAuthentication` is also an exported method. Not sure why that would be, and if the same reasons would apply to `ConfigureCompression`?

Reply:
Yeah I agree it doesn't need to - will do!

Reply:
> I also don't see why this would be exported, but I saw that `ConfigureAuthentication` is also an exported method. Not sure why that would be, and if the same reasons would apply to `ConfigureCompression`?

ConfigureAuthentication is exported because it is called by the Kafka receiver. I think configureCompression doesn't need to at this point.